### ISSUE TITLE
Timeout during composer update

### ISSUE THREAD

**Issue by CmdrSharp** [travis-ci/travis-ci#9449]
I've had jobs get stuck during the "composer update"-section of builds. Several of these have worked on PHP 7.1, but have gotten stuck when doing the same thing for 7.2. I've tried re-running individual jobs several times to no avail.

Currently stuck jobs:
https://travis-ci.org/CmdrSharp/amqp-route-messenger/jobs/362545831
https://travis-ci.org/CmdrSharp/amqp-route-messenger/jobs/362545832

I figured this was related to yesterdays [incident](https://www.traviscistatus.com/incidents/b8p2nwv4spcr), but that has been marked resolved and the issues continue.

**Comment by BanzaiMan** [comment]
Do the dependencies resolve in reasonable time on your machine? I tried it on mine, and it seems to take a long time as well. Could you try reproducing the problem with our Docker image? https://docs.travis-ci.com/user/common-build-problems/#Troubleshooting-Locally-in-a-Docker-Image

**Comment by CmdrSharp** [comment]
@BanzaiMan I did as you suggested and attempted the build on a docker installation. Same issue.
I did so with verbose Composer output (see the log file [with-require-dev-all.txt](https://github.com/travis-ci/travis-ci/files/1881425/with-require-dev-all.txt)), but it basically gets stuck here:
"Resolving dependencies through SAT"

Not sure that tells me very much however. What I did as a debugging step was remove the require-dev section of the composer.json in the build. It then had no problems installing (see [no-require-dev.txt](https://github.com/travis-ci/travis-ci/files/1881423/no-require-dev.txt)).

Finally, I did a pull with only phpunit as the require-dev. This also works ([with-require-dev-phpunit.txt](https://github.com/travis-ci/travis-ci/files/1881424/with-require-dev-phpunit.txt)).
So it appears it fails when requiring orchestral.

**But**, the question remains why, though. I can fetch it from GitHub, and I had no issues pulling orchestral in when requiring it normally (see [require-orchestral.txt](https://github.com/travis-ci/travis-ci/files/1881422/require-orchestral.txt)).

_Note: I cleared composer cache in between each attempt just to be safe._

Any hints? I'm out in unknown territory here with no clue as to what I'm missing, especially since the 7.1 builds work (though take a long time).
